query/objects/network_endpoint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "caption": "Network Endpoint",
+  "name": "network_endpoint",
+  "extends": [null, "network_endpoint"],
+  "observable": 0,
```

Comment:
Why was this added, was it intentional? You might want to check if this affected if `Observables` now has an `Observables.network_endpoint` after running a code generator, or by looking at it from the `Schema` class.

If it doesn't, that's probably because we had a test like `if record.get("observable"): ...` and `0` causes that branch to be skipped.

Reply:
I basically copied this from `network_endpoint.json` in the root namespace.